pandas/tools/tests/test_util.py

```diff
@@ -291,6 +291,70 @@ def test_non_hashable(self):
         with self.assertRaisesRegexp(TypeError, "Invalid object type"):
             pd.to_numeric(s)
 
+    def test_compact_ints(self):
+        s = ['1', 2, 3]
+
+        res = pd.to_numeric(s)
+        expected = np.array([1, 2, 3], dtype=np.int64)
+        tm.assert_numpy_array_equal(res, expected)
+
+        res = pd.to_numeric(s, compact_ints=True)
+        expected = np.array([1, 2, 3], dtype=np.int8)
+        tm.assert_numpy_array_equal(res, expected)
+
+        res = pd.to_numeric(s, compact_ints=True, use_unsigned=True)
+        expected = np.array([1, 2, 3], dtype=np.uint8)
+        tm.assert_numpy_array_equal(res, expected)
+
+        # ignore use_unsigned (negative number)
```

Comment:
nothing to do with ``read_csv``, but I asked it to ``compact_ints``. 

ok, well add a test anyhow that it passes thru the data ok then.